apps/cookbook/src/app/showcase/card-showcase/card-showcase.component.ts

```diff
@@ -31,10 +31,18 @@ export class CardShowcaseComponent {
       defaultValue: '',
       type: ['string'],
     },
+    {
+      name: 'variant',
+      description:
+        'Sets the variant of a card. The recommended is "elevated" which is also the default',
+      defaultValue: 'elevated',
+      type: ['elevated', 'flat', 'outlined'],
```

Comment:
Consider keeping naming in sync with UX:
```suggestion
      type: ['elevated', 'flat', 'outline'],
```